src/utils/loadScripts.py

```diff
@@ -1,180 +1,362 @@
 ########### Import Libraries
 import logging, verboselogs
+
 logger = verboselogs.VerboseLogger(__name__)
 from dateutil.relativedelta import relativedelta as rd
-import requests 
-from src.utils.rest import rest_get, rest_post # 
+import requests
+from src.utils.rest import rest_get, rest_post  #
 import asyncio
 from packaging import version
 
+import logging
+from datetime import datetime
+import pytz  # Requires installing the pytz module
+
+
+class TZFormatter(logging.Formatter):
+    def __init__(self, tz_name, fmt=None, datefmt="%Y-%m-%d %H:%M:%S"):
+        super().__init__(fmt, datefmt)
+        self.tz = None
+        if tz_name:
+            try:
+                self.tz = pytz.timezone(tz_name)
+            except pytz.UnknownTimeZoneError:
+                logging.error(
+                    f"Invalid timezone specified: {tz_name}. Time will not be used."
+                )
+
+    def formatTime(self, record, datefmt=None):
+        if self.tz:
+            dt = datetime.fromtimestamp(record.created, self.tz)
+            return dt.strftime(self.datefmt)
+        return super().formatTime(record, datefmt)
+
+
 def setLoggingFormat(settingsDict):
-    # Sets logger output to specific format
-    log_level_num=logging.getLevelName(settingsDict['LOG_LEVEL'])
```

Comment:
If we included the specific setting that defines whether to print out or not the timestamp, I think all we had to do is to add a if/else statement here to include it, wouldn't we?

Reply:
I think that timestamps are an expected compliment to logs globally. 

I can not foresee anyone complaining of timestamps being incorporated into system logs and I'm not sure if it's worth introducing a toggle feature?

If this is dealbreaker however, I'm happy to accomodate this request.

Reply:
> I can not foresee anyone complaining of timestamps being incorporated into system logs and I'm not sure if it's worth introducing a toggle feature?

when you look at the logs through dozzle, the timestamp is already there. Thus if we added it as you suggest (without toggle), wouldn‘t the timestamp show twice?